junit-jupiter-engine/src/test/java/org/junit/jupiter/api/AssertionsAssertAllTests.java

```diff
@@ -149,7 +149,7 @@ void assertAllWithExecutableThatThrowsBlacklistedException() {
 	}
 
 	@SafeVarargs
-	private static void assertExpectedExceptionTypes(MultipleFailuresError multipleFailuresError,
+	static void assertExpectedExceptionTypes(MultipleFailuresError multipleFailuresError,
```

Comment:
This method is incredibly useful and I would 100% support making it part of the public API.